chain-impl-mockchain/src/fragment/test.rs

```diff
@@ -18,8 +18,7 @@ impl Arbitrary for Fragment {
             7 => Fragment::PoolUpdate(Arbitrary::arbitrary(g)),
             8 => Fragment::UpdateProposal(Arbitrary::arbitrary(g)),
             9 => Fragment::UpdateVote(Arbitrary::arbitrary(g)),
-            10 => Fragment::VotePlan(Arbitrary::arbitrary(g)),
-            11 => Fragment::VoteCast(Arbitrary::arbitrary(g)),
```

Comment:
potentially missing access to testing the plan and tally certificates

Reply:
added missing `Arbitrary` impls